core/src/main/scala/org/http4s/jdkhttpclient/JdkHttpClient.scala

```diff
@@ -71,8 +71,7 @@ object JdkHttpClient {
             req.entity match {
               case Entity.Empty => BodyPublishers.noBody()
               case Entity.Strict(chunk) =>
-                val slice = chunk.toArraySlice
-                BodyPublishers.ofByteArray(slice.values, slice.offset, slice.length)
+                BodyPublishers.ofInputStream(() => chunk.toInputStream)
```

Comment:
@armanbilge FYI: this addresses https://github.com/http4s/http4s-jdk-http-client/pull/644#discussion_r882671003

Reply:
Hooray thank you! Just a nit, I would rename `chunk` -> `bytes` or something :)